lib/testing_utils.py

```
@@ -0,0 +1,89 @@
+    from rich.console import Console
+    from rich.table import Table
+    RICH_AVAILABLE = True
+except ImportError:
+    RICH_AVAILABLE = False
+
+
+T = TypeVar("T")
+P = ParamSpec("P")
+R = TypeVar("R")
+
+def track_runtime(should_print: bool) -> Callable[[Callable[P, R]], Callable[P, R]]:
```

Ruff (0.15.7)

[warning] Boolean-typed positional argument in function definition

(FBT001)